201-vm-push-certificate-windows/azuredeploy.json

```diff
@@ -71,20 +71,21 @@
     "imageOffer": "WindowsServer",
     "imageSKU": "2012-R2-Datacenter",
     "publicIPName": "certPublicIP",
-    "publicIPAddressType": "Dynamic"
+    "publicIPAddressType": "Dynamic",
+    "apiVersion": "2015-06-15"
```

Comment:
Don't use a variable for apiVersion, it breaks schema validation.  Just update to the latest.